CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 
+* Update PowerSTIG to successfully parse/apply Mozilla Firefox - V4R29: [698](https://github.com/microsoft/PowerStig/issues/698)
```

Comment:
Should be a pound sign before the number in "698", I've fixed the one below it in one of my PRs recently.